docs/webui.md

```diff
@@ -225,25 +233,55 @@ Creating a catalog requires these fields:
   </TabItem>
 </Tabs>
 
+:::tip
+Due to the current limitation of the web interface, which only allows for viewing, the functionality to create or modify schema, tables, or filesets is not available. Please refer to the [documentation](./manage-fileset-metadata-using-gravitino.md) to use the REST API for these operations.
+:::
+
+###### 2. Type `fileset`
+
+<Tabs>
+  <TabItem value='hadoop' label='Hadoop'>
+    Follow the [Hadoop catalog](./hadoop-catalog.md) document.
+
+    <Image img={require('./assets/webui/create-fileset-hadoop-catalog-dialog.png')} style={{ width: 480 }} />
+
+  </TabItem>
+</Tabs>
+
+###### 3. Type `messaging`
+
+<Tabs>
+  <TabItem value='kafka' label='Kafka'>
+    Follow the [Kafka catalog](./kafka-catalog.md) document.
```

Comment:
It seems there are currently no Kafka-related documents, so we need to wait for the related PR to be merged.